docs/docs/tutorials/using_dask.md

```diff
@@ -110,20 +122,78 @@ On executing the cell, you'll see the following:
+Dask's browser-based dashboard opens automatically.
 :::
 
-1. To get started, we will load the data using a Dask DataFrame. This will lazily load the dataset. Add the following to your notebook:
+You can access the dashboard in two ways:
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+  <TabItem value="browser-based" label="Browser-based diagnostic dashboard" default>
+
+Dask's diagnostic dashboard will open in a new browser tab if you click on the dashboard links displayed in the Client UI or Cluster UI above.
+
+![Dask diagnostic UI - showing four parallel computation streams](/img/tutorials/dask_diagnostic_dashboard.png)
+
+  </TabItem>
+  <TabItem value="labextension" label="Dask JupyterLab extension (recommended)">
+
+The [Dask-labextension](https://github.com/dask/dask-labextension) is included with the default Nebari deployment.
+It provides a JupyterLab extension to manage Dask clusters,
+as well as embed Dask's dashboard plots directly into JupyterLab panes.
+
+On JupyterLab, you can select the Dask icon in the left sidebar and click on the magnifying glass icon to view the list of available plots.
+Alternatively, you can copy-and-paste the dashboard link (displayed in the Client UI) in the input field.
+
+![Dask-labextension UI In JupyterLab - displays the demo notebook and a list of cluster settings](/img/tutorials/dask_labextension.png)
+
+  </TabItem>
+</Tabs>
+
+Let's open and understand the dashboard plots `Task Stream`, `Progress`, and `Cluster map`.
+
+Most colors and the interpretation would differ based on the computation you choose.
+However, the colors remain consistent and represent the same task across all plots within the dashboard.
+Note that some colors always have the same meaning, like red always indicates inter-worker communication and data transfer.
+
+Each of the computation you submit to Dask (you will learn more in the next section) is split into multiple tasks for parallel execution.
+
+In the `Progress` plot, we see the distinct colors associated with different tasks to complete the overall computation.
+
+In the `Task Stream` (a streaming plot) each row represents a thread (a Dask worker can have multiple threads)
+and the small rectangles within are the individual tasks.
+
+The `Cluster Map` shows the Dask scheduler at the center in a purple circle, with the active Dask workers around it in yellow circles.
+This diagram is a convenient way to visualize Adaptive scaling, and ensure new workers are being spun-up and shut-down based on workflow demand.
+
+Check out the [Dask Documentation on the dashboard plots](https://docs.dask.org/en/stable/dashboard.html) for more information.
+Keep the dashboard plots open for the following computations!
+
+## Step 5 - Run your computation with Dask
+
+In the following sections, you will perform some basic analysis on the well-known New York City yellow taxi dataset,
```

Comment:
We need a link to the original source behind "New York City yellow taxi dataset"

Reply:
Done!